drag-arrange.js

```diff
@@ -84,9 +81,11 @@
       var $this = $(this);
 
       if (dragSelector) {
-        $this.on(dragEvents.START + eventNamespace, dragSelector, dragStartHandler);
+        $this.on(dragEventsMouse.START + eventNamespace, dragSelector, dragStartHandler);
```

Comment:
@zzgab will it not bind 2 times for touch screen devices? Just curious! haven't tested.